<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(255, 161, 107);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QTextEdit" name="input">
      <property name="styleSheet">
       <string notr="true">border: 10px;
background-color: rgb(255, 224, 196);
color: rgb(50, 50, 50);
border-radius: 20px ;</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="btn_copy_to_clipboard">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 10px;
background-color: rgb(255, 210, 120);
color: rgb(50, 50, 50);
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string>Copy to ClipBoard</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="l_breaker">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 184, 43);</string>
      </property>
      <property name="text">
       <string>Remove line breaks only</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="p_breaker">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 184, 43);</string>
      </property>
      <property name="text">
       <string>Remove line breaks and paragraph breaks</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QTextEdit" name="output">
      <property name="styleSheet">
       <string notr="true">border: 10px;
background-color: rgb(255, 224, 196);
color: rgb(50, 50, 50);
border-radius: 20px ;</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="btn_line_breaker">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 10px;
background-color: rgb(255, 210, 120);
color: rgb(50, 50, 50);
border-radius: 10px ;</string>
      </property>
      <property name="text">
       <string>Remove Line Breaks</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 184, 43);</string>
      </property>
      <property name="text">
       <string>New Text without Line Breaks</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="titr">
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 10px;
background-color: rgb(255, 210, 120);
color: rgb(50, 50, 50);
border-radius: 20px ;</string>
      </property>
      <property name="text">
       <string>   Remove Line Breaks</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
